src/lib/journey/callbacks/recaptcha/recaptcha.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+  import type { ReCaptchaCallback } from '@forgerock/javascript-sdk';
+  import type { z } from 'zod';
+  import { onMount } from 'svelte';
+
+  import type { SelfSubmitFunction, StepMetadata } from '$journey/journey.interfaces';
+  import type { styleSchema } from '$lib/style.store';
+  import type { Maybe } from '$lib/interfaces';
+
+  export let callback: Maybe<ReCaptchaCallback>;
+  export const selfSubmitFunction: Maybe<SelfSubmitFunction> = null;
+  export const stepMetadata: Maybe<StepMetadata> = null;
+  export const style: z.infer<typeof styleSchema> = {};
+
+  const siteKey = callback?.getSiteKey() ?? '';
+  let recaptchaApi = `${callback?.getOutputByName<string>('captchaApiUri', '') ?? ''}`;
+  /**
+   * AM defaults the class name to g-captcha which is wrong
```

Comment:
"wrong" sounds a bit harsh :)

Reply:
i can make it "incorrect" if that sounds better 👍